README.md

```diff
@@ -91,6 +91,11 @@ make install
 ```
 
 
+## Contributing
+
+If you would like to contribute to the OpenSCAP, either fix some issue or add a new feature, please check our [contribution guide](docs/contribute/contribute.adoc).
```

Comment:
> **you can** fix some issue or add a new feature

I suggest to be less strict and more open.

We could also invite people to join the mailing list here and contribute to discussions. 
Only mention of the list I could find was in section `Choose the right branch`